src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -1313,6 +1313,21 @@ - (void)invokeCommandWithEndpointID:(NSNumber *)endpointID
     // here for now.
     // TODO: https://github.com/project-chip/connectedhomeip/issues/24563
 
+    if (![commandFields isKindOfClass:NSDictionary.class]) {
```

Comment:
Will this change break any clients that might have set this to nil (even though it says it not nullable)

Reply:
Yes, it will, but I would have expected those clients to fail to compile....  That said, we could do the same thing the no-timedInvokeTimeout overload does to handle nil here.

Reply:
Actually, no, this never worked: down lower when we try to convert the arg to TLV, we end up with an error if it's nil.  So now we're ending up with exactly the same error, but a bit earlier up the callstack.